aten/CMakeLists.txt

```diff
@@ -135,6 +135,9 @@ list(APPEND ATen_CPU_INCLUDE
   ${CMAKE_CURRENT_BINARY_DIR}/src/ATen)
 add_subdirectory(src/ATen)
 
+file(GLOB ATen_CORE_TEST_SRCS "*_test.cpp")
+EXCLUDE(ATen_CPU_SRCS "${ATen_CPU_SRCS}" ${ATen_CORE_TEST_SRCS})
```

Comment:
This looks like copy-paste. Can't you propagate it from aten/src/ATen/CMakeLists.txt by having that cmake set it to PARENT_SCOPE?

Reply:
fixed